specs/commandline/policy.md

```diff
@@ -0,0 +1,232 @@
+
+Flags:
+  -d, --debug     debug mode
+  -h, --help      help for show
+  -v, --verbose   verbose mode
+```
+
+## Usage
+
+### Initialize trust policies with default values
+
+```shell
+notation policy init
+```
+
+Upon successful execution, trust policies with default values are created and printed out as following. Use command `notation cert add --type ca --store default <cert_file>` to add CA certificates to trust store `ca:default`. If users want to use different trust stores, refer to section [Update trust policies](#update-trust-policies) on how to do the update.
+
+```jsonc
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "policy-by-init-command",
+            "registryScopes": ["*"],
+            "signatureVerification": {
+                "level": "strict"
+            },
+            "trustStores": ["ca:default"],
+            "trustedIdentities": ["*"]
+        }
+    ]
+}
+
+```
+
+If there are existing trust policies configured and users still run `notation policy init` command, A prompt should be displayed asking for confirmation on whether restoring to default values or not. Use `--force` flag to discard any existing trust policies without prompt.
+
+### Initialize trust policies with specified trust stores
+
+```shell
+notation policy init --ts "ca:my_store" --ts "ca:my_store_2"
+```
+
+Upon successful execution, trust policies with default values and specified trust stores are created and printed out as following:
+
+```jsonc
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "policy-by-init-command",
+            "registryScopes": ["*"],
+            "signatureVerification": {
+                "level": "strict"
+            },
+            "trustStores": ["ca:my_store", "ca:my_store_2"],
+            "trustedIdentities": ["*"]
+        }
+    ]
+}
+```
+
+Use `--force` flag to override existing policies without prompt.
+
+### Show trust policies
+
+```shell
+notation policy show
+```
+
+Upon successful execution, the trust policies are printed out. If trust policies are not configured, users should receive an error message, and a tip to initialize trust policies using command `notation policy init`.
+
+### Export trust policies into a JSON file
+
+Users can redirect the output of command `notation policy show` to a JSON file.
+
+```shell
+notation policy show > ./trust_policy.json
+```
+
+### Import trust policies from a JSON file
+
+```shell  
+notation policy import ./my_policy.json
+```
+
+The trust policies in the JSON file will be validated according to [trust policy properties](https://github.com/notaryproject/notaryproject/blob/v1.0.0-rc.2/specs/trust-store-trust-policy.md#trust-policy-properties). A successful message should be printed out if trust policies are imported successfully. Error logs including the reason should be printed out if the importing fails.
+
+Use `--force` flag to override existing policies without prompt.
```

Comment:
Question: What if the imported policy file is invalid? Will the `--force` flag let it import?

Reply:
The import will fail if the JSON file is invalid. it should be validated before overwriting the existing one.